packages/sdk/src/protocols/base_protocol.ts

```diff
@@ -227,10 +222,17 @@ export class BaseProtocolSDK implements IBaseProtocolSDK {
   private async findAdditionalPeers(numPeers: number): Promise<Peer[]> {
     this.log.info(`Finding ${numPeers} additional peers`);
     try {
-      let newPeers = await this.core.allPeers();
+      let newPeers = await this.core.getPeers({
```

Comment:
did you try this in practice, e.g dogfooding app? 
I used `allPeers` because `getPeers` were not returning everything, and was limited for some reason